spec.html

```diff
@@ -4471,20 +4472,20 @@ <h1>Data Blocks</h1>
 
       <emu-clause id="sec-createbytedatablock" aoid="CreateByteDataBlock">
         <h1>CreateByteDataBlock ( _size_ )</h1>
-        <p>The abstract operation CreateByteDataBlock takes argument _size_ (an integer). It performs the following steps when called:</p>
+        <p>The abstract operation CreateByteDataBlock takes argument _size_ (an integral Number). It performs the following steps when called:</p>
         <emu-alg>
-          1. Assert: _size_ &ge; 0.
+          1. Assert: _size_ &ge; *+0*.
           1. Let _db_ be a new Data Block value consisting of _size_ bytes. If it is impossible to create such a Data Block, throw a *RangeError* exception.
-          1. Set all of the bytes of _db_ to 0.
+          1. Set all of the bytes of _db_ to *+0*.
```

Comment:
It's better to _internally_ keep both the size and the byte values as integers, not integral Numbers. This avoids problems with ±0 and incorrect behavior when _size_ > 2<sup>53</sup>. See my next comment for why.

Reply:
I think we need to reach a consensus here. It is clashing with @michaelficarra's previous request to remove subscripts for `*+0*` and `*-0*`. I don't have a strong opinion about it and I'm fine with both directions, but it would be good to have a clear path to follow.